spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -1,14 +1,39 @@
 describe ManageIQ::Providers::AnsibleTower::AutomationManager::Refresher do
-  let(:auth)                    { FactoryGirl.create(:authentication) }
+  # To re-record cassettes or to add cassettes you can add another inner `VCR.use_cassette` block to the
```

Comment:
@djberg96 @bronaghs @Ladas @bdunne @blomquisg @Fryguy 

I think this is a neat way for building up vcr cassettes (aka tape library) without pet sitting those existing cassettes. This might break with interdependent requests.